### DISCUSSION TITLE
Usability - Make the name of games easier to select on game pages

### DISCUSSION THREAD

**Discussion by Crefossus** [ludumdare/ludumdare#2126] [Ideas]
I think the solution is not to make the name a link. However I don't understand why it's a link in the first place so I'm probably wrong. 

Normally you can just double/triple click anywhere on text in h1, etc to select it. 

Since the title of a game is a link it's a bit tedious/finicky to select (firefox at least). I've gone into dev tools to select the text for longer titles or when I couldn't get the highlight working. Specifically I mean the text inside inside the div "content-common-body -title _font2", which is the name of the game.

Wikipedia doesn't seem to have their page h1 as links for example (even on mobile).

**Comment by mikekasprzak** [comment]
Hmm. Yeah, that behavior is a bit of a side effect of everything _under the hood_ being a "node", which initially meant everything was a blog post, and you had to click on post titles (or the comments icon) to see comments.

It's an interesting observation that copying a name is something you might want to do, especially when only a single node is the focus. 🤔
